src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
+jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist zu voll
+jei.tooltip.error.recipe.transfer.unknown=Unbekannter Fehler, siehe Logs
+jei.tooltip.error.recipe.transfer.no.server=Auf dem Server muss JEI installiert sein
 
 # Error Messages
-jei.chat.error.command.too.long=JEI muss auf dem Server installiert sein. um mit diesem Befehl klarzukommen, da er zu lang für Minecraft ist.
+jei.chat.error.command.too.long=JEI muss auf dem Server installiert sein, damit dieser Befehl funktioniert, da er zu lang für Minecraft ist.
 
 # Key Bindings
-key.jei.toggleOverlay=Item-Listen-Overlay (de-)aktivieren
-key.jei.showRecipe=Zeige Item-Rezepte
-key.jei.showUses=Zeige Item-Nutzung
+key.jei.toggleOverlay=Gegenstandsliste (de-)aktivieren
+key.jei.focusSearch=Suchfeld
+key.jei.showRecipe=Zeige Rezept
+key.jei.showUses=Zeige Verwendung
+key.jei.recipeBack=Zeige vorherige Rezeptseite
+key.jei.toggleCheatMode=Gegenstands-Cheating (de-)aktivieren
 
 # Config
 config.jei.default=Standard
 config.jei.valid=Zulässig
 
-config.jei.title=%MODNAME Konfiguration
+config.jei.title=%MODNAME-Konfiguration
 
 config.jei.mode=Modus
 config.jei.mode.comment=Ändere den Modus, in dem sich JEI befindet
-config.jei.mode.cheatItemsEnabled=Aktiviere Item-Cheating
-config.jei.mode.cheatItemsEnabled.comment=Gibt Items, statt die Rezepte zu zeigen.
-config.jei.mode.editEnabled=Aktiviere Item-Listeneditiermodus
-config.jei.mode.editEnabled.comment=Verstecke und zeige Items durch Klicken in der Item-Liste.
+config.jei.mode.cheatItemsEnabled=Aktiviere Gegenstands-Cheating
```

Comment:
You're following rules that allow to transform a language just because of opinion. You don't have the choice you want to have. You don't own the language. You should take into account what others think and what rule softening can lead to. You can do so, but then you should translate for yourself instead of forcing your opinion to others.